### ISSUE TITLE
VMRay Service

### ISSUE THREAD

**Issue by cccs-rs** [CybercentreCanada/assemblyline#56]
**Is your feature request related to a problem? Please describe.**
Has been requested over the years but there is no official service because of the license requirements & lack of API documentation:
[VMRay integration (google.com)](https://groups.google.com/g/cse-cst-assemblyline/c/sHPsL6qYuow/m/eex8ZjeXBAAJ?utm_medium=email&utm_source=footer)

**Describe the solution you'd like**
Community-written service to leverage VMRay analysis

**Comment by ed4wg** [comment]
@eljeffeg - we're also interested in integrating with VMRay? Any luck on this service?

**Comment by eljeffeg** [comment]
Yes, but we're dealing with issues with VMRay itself.  The dynamic piece isn't working as the win11 vms are failing to start.  Once we get that all figured out, we'll continue working on the AL service.

**Comment by mback2k** [comment]
@eljeffeg any update on your side? We also have a VMRay instance and would like to see a service developed. Before we start from scratch, maybe we can join forces?

**Comment by eljeffeg** [comment]
@ed4wg @mback2k We haven't been able to finish it as we've been busy with other things, but here is what we have thus far, which should at least pull back results and such.  Happy to join forces and get it done.  https://github.com/fbicyber/assemblyline-service-vmray
